### ISSUE TITLE
Old reference of callbackWrapper in use after reconnect

### ISSUE THREAD

**Issue by knalli** [knalli/angular-vertxbus#55]
fix todo

```
          if (connectionState === vertxEventBus.EventBus.OPEN) {
            util.registerHandler(address, callback);
            unregisterFn = function () {
              util.unregisterHandler(address, callback)
            };
          }

```

**Comment by knalli** [comment]
fyi: @leolux available in 1.1.3

**Comment by leolux** [comment]
I've updated the reproducer https://github.com/leolux/angular-vertxbus_removelistener to version 1.1.3. and everything works perfectly well now. 
Thank you knalli!
